frontend/containers/open-call-page/header/component.tsx

```diff
@@ -0,0 +1,190 @@
+import { FC, useMemo } from 'react';
+
+// import { Heart } from 'react-feather';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import cx from 'classnames';
+
+import { useRouter } from 'next/router';
+
+import dayjs from 'dayjs';
+
+import { translatedLanguageNameForLocale } from 'helpers/intl';
+
+import Breadcrumbs from 'containers/breadcrumbs';
+import ShareIcons from 'containers/share-icons';
+
+// import Button from 'components/button';
+// import Icon from 'components/icon';
+import LayoutContainer from 'components/layout-container';
+import { OpenCallStatus } from 'enums';
+
+import OpenCallChart from '../chart';
+
+import { OpenCallHeaderProps } from '.';
+
+export const OpenCallHeader: FC<OpenCallHeaderProps> = ({
```

Comment:
This section is missing, right?
![Open call description and investor profile picture](https://user-images.githubusercontent.com/6073968/185358745-e8b91fb3-d4df-4f69-970f-89da462d34d4.png)

Reply:
We don't have anything to show here. The description is being displayed on the overview